students/tri_nguyen/lesson03/list_lab.py

```diff
@@ -0,0 +1,179 @@
+# Instructor: Natasha Aleksandrova
+# Date:  09-Feb-2018
+# ------------------------------------------- #
+
+# ------------------------------------------- #
+# Series 1:
+# - Create a list that contains 'Apples', 'Pears',
+#   'Oranges' and 'Peaches.
+# - Display the list.
+# - Ask the user for another fruit and add it
+#   to the end of the list.
+# - Display the list.
+# - Ask the user for a number and display the number
+#   back to the user and the fruit corresponding
+#   to that number.
+# - Add another fruit to the beginning of the list
+#   using '+' and display the list.
+# - Add another fruit to the beginning of the list
+# - using insert() and display the list.
+# - Display all the fruits that begins with 'P',
+#   using a for loop.
+
+
+fruit_list = ['Apples', 'Pears', 'Oranges', 'Peaches']
+
+print('--Series 1 of the exercise--'.upper()+'\n')
+
+print('Below is the current list of fruits.')
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+user_input = input('Please add another fruit: ')
+fruit_list.append(user_input.title())
+
+print()
+print('Below is a newly updated list of fruits.')
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+num_from_user = int(input('Please pick a number to get back the corresponding fruit: '))
+num_from_user = num_from_user - 1  # Python uses zero based indexing
+
+if fruit_list[num_from_user] in fruit_list[num_from_user]:
+    print(num_from_user + 1, '-', fruit_list[num_from_user])
+
+print()
+fruit_list = ['Watermelons'] + fruit_list
+print('Watermelons has been added to the beginning of the list.')
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+fruit_list.insert(0, 'Pinnaples')
+print('Pinnaples have been added to the beginning of the list.')
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+print('Below is the list of fruits that begin with "P".')
+for fruit in fruit_list:
+    if fruit.startswith('P'):
+        print(fruit)
+
+# End of Series 1
+# ------------------------------------------- #
+
+# ------------------------------------------- #
+# Series 2 - Using the list created in series 1:
+# - Display the list.
+# - Remove the last fruit from the list.
+# - Display the list.
+# - Ask the user for a fruit to delete, find it
+#   and delete it.
+# - (Bonus: Multiply the list times two. Keep asking
+#    until a match is found. Once found, delete all
+#    occurrences).
+
+print('\n'+'--Series 2 of the exercise--'.upper()+'\n')
+
+print('Below is the current list of fruits.')
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+print('The last fruit ({}) from the list has been removed.'.format(fruit_list[len(fruit_list) - 1]))
+fruit_list.pop()
+for idx, fruit in enumerate(fruit_list):
+    print(idx + 1, '-', fruit)
+
+print()
+to_be_deleted = input('Enter a fruit to delete: ')
+
+if to_be_deleted.title() in fruit_list:
```

Comment:
you can always do shorthand: `input('Enter a fruit to delete: ').title()` so that you don't have to repeat `title()` call